src/test/java/co/rsk/federate/signing/utils/TestUtils.java

```diff
@@ -89,11 +94,26 @@ public static Block mockBlockWithUncles(long number, Keccak256 hash, long diffic
     public static Federation createFederation(NetworkParameters params, int amountOfMembers) {
         List<BtcECKey> keys = Stream.generate(BtcECKey::new).limit(amountOfMembers).collect(Collectors.toList());
         List<FederationMember> members = FederationMember.getFederationMembersFromKeys(keys);
-        FederationArgs federationArgs = new FederationArgs(members, Instant.now(), 0, params);
+        FederationArgs federationArgs = new FederationArgs(members, Instant.now(), CREATION_BLOCK_NUMBER, params);
+
+        return FederationFactory.buildStandardMultiSigFederation(federationArgs);
+    }
 
+    public static Federation createFederation(NetworkParameters params, List<BtcECKey> federationPrivatekeys) {
```

Comment:
Is this used?

Reply:
That's correct! It is used in some tests. It's part of the solution required to avoid using `GenesisFederation`.

Reply:
Sure? I don't see any uses of it

Reply:
is being used in [here](https://github.com/rsksmart/powpeg-node/blob/3ea9942c4d4e38a372d31da5a8aa64d75b86471b/src/test/java/co/rsk/federate/BtcToRskClientTest.java#L109) 

and [here](https://github.com/rsksmart/powpeg-node/blob/3ea9942c4d4e38a372d31da5a8aa64d75b86471b/src/test/java/co/rsk/federate/bitcoin/BitcoinWrapperImplTest.java#L192)

Reply:
As Julia said, it is used in those tests.

Reply:
My bad, sorry. Intellij was showing it as not used but it was wrongly configured